pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1224,7 +1224,6 @@ with haskellLib;
   Cabal-ide-backend = dontCheck super.Cabal-ide-backend;
 
   # This package can't be built on non-Windows systems.
-  Win32 = overrideCabal (drv: { broken = !pkgs.stdenv.hostPlatform.isCygwin; }) super.Win32;
```

Comment:
According to https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/libraries/version-history this has long been a boot package on windows. Since we can't build it elsewhere anyway I left it as `Win32 = null;`